supervisor/api/middleware/security.py

```diff
@@ -12,7 +12,7 @@
     ROLE_DEFAULT,
     ROLE_HOMEASSISTANT,
     ROLE_MANAGER,
-    CoreState,
+    SupervisorState,
```

Comment:
We already have `sys_supervisor` for <https://github.com/home-assistant/supervisor/blob/main/supervisor/supervisor.py#L35>

Reply:
:grimacing: uh, that would need some naming decision then...

Maybe sticking with the names and improve the doc strings somewhat is the better way to go:

```diff
class CoreState(str, Enum):
-    """Represent current loading state."""
+    """Represent current Supervisor state."""
```

Reply:
Well, homeassistant is still called homeassistant inside supervisor. The `Core` represents the core of the system. We would never rename `sys_homeassistant`. If we make an object that represents the Home Assistant Core State, it would still be called `HomeAssistantState` or so.  I will think about it and iterate on it next time I will work on the supervisor, maybe not this year.

Reply:
we should not update our comments in that way ;) make the next comment hard to understand. I think renaming the doc-string is enough. I was even not a fan of renaming it on API because `homeassistant` is more representative as `core` and each backend software has its own core.